middlewares/hacker.middleware.js

```diff
@@ -534,6 +568,18 @@ function parseAccept(req, res, next) {
     next();
 }
 
+/**
+ * Sets req.body.status to Accepted for next middleware.
+ * @param {{params:{email: string}, body: *}} req 
+ * @param {*} res 
+ * @param {*} next 
+ */
+function parseAcceptEmail(req, res, next) {
```

Comment:
No need to make another function for parseAcceptEmail if its also going to set req.body.status to accepted.